.vib/grafana-tempo/vib-publish.json

```diff
@@ -0,0 +1,68 @@
+{
+  "phases": {
+    "package": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/grafana-tempo"
+        }
+      },
+      "actions": [
+        {
+          "action_id": "helm-package"
+        },
+        {
+          "action_id": "helm-lint"
+        }
+      ]
+    },
+    "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/grafana-tempo"
+        },
+        "runtime_parameters": "InF1ZXJ5RnJvbnRlbmQiOgogICJzZXJ2aWNlIjoKICAgICJwb3J0cyI6CiAgICAgICJodHRwIjogODAKICAgICJ0eXBlIjogIkxvYWRCYWxhbmNlciI=",
+        "target_platform": {
+          "target_platform_id": "{VIB_ENV_TARGET_PLATFORM}",
+          "size": {
```

Comment:
@joancafom Is this size compatible with all target-platforms?

Reply:
It is compatible with all the `target-platforms` we've used so far, namely: TKG, AKS, GKE, and EKS. I did not check others.